### DISCUSSION TITLE
using this tool to get Green-Ampt parameters to estimate infiltration

### DISCUSSION THREAD

**Discussion by aniroomandi** [ar-siddiqui/curve_number_generator#36] [Ideas]
Hi,
I just used this tool to see how things work and so far it seems everything is looking great by just looking at the generated outputs. So thank you very much for creating such a great tool.

I wanted to share this idea to see the possibility of outputting the Green-Ampt infiltration parameters as well with this tool.
Green-Ampt methodology is also one of the popular approaches when it comes to calculating Hydrology parameters mostly in HECHMS or SWMM models. I was wondering if you have any plans to obtain the Green-Ampt parameters from Soil website. The key to get the related parameters are to identify the soil in such category:
Sand,Loamy Sand,Sandy Loam, Loam 3.50, Silt Loam, Sandy Clay Loam, Clay Loam, Silty Clay Loam, Sandy Clay, Silty Clay, Clay
Then using these soils the required input parameters can be extracted using this table:
https://files.knowyourh2o.com/Waterlibrary/Stormwater/greenamp.pdf

Please let me know if you would like to discuss this further.

Thanks again for proving this great tool and looking forward to discussing the above further with you.

**Comment by ar-siddiqui** [comment]
Hi,
Can you provide a simple step-by-step workflow on how Green Ampt Layer will be generated from soils?

**Comment by aniroomandi** [comment]
Hi Abdul,

thanks for getting back to me on this. Yes there is a direction provided by
the USACE to get the Green-Ampt inputs from the Soil Survey, please google
below

"Analysis of SURRGO Data and Obtaining Soil Texture Classifications for
Simulating Hydrologic Processes"

Or download the attachment. Please let me know if you want to discuss this
further.

Thanks
Arash
